src/main/resources/db/migration/V20210322_16.2__CPO-16-CasePaymentOrdersAUD.sql

```diff
@@ -0,0 +1,39 @@
+CREATE SEQUENCE hibernate_sequence INCREMENT 1 MINVALUE 1
+    MAXVALUE 9223372036854775807
+    START 1
+    CACHE 1;
+
+CREATE TABLE public.revinfo
+(
+    rev      integer not null constraint revinfo_pkey primary key,
+    revtstmp bigint
+);
+
+CREATE TABLE public.case_payment_orders_audit
+(
+    id                    uuid    not null,
+    rev                   integer not null,
+    revtype               smallint,
+    action                varchar(70),
+    action_mod            boolean,
+    case_id               bigint,
+    case_id_mod           boolean,
+    created_by            varchar(70),
+    created_by_mod        boolean,
+    created_timestamp     timestamp,
```

Comment:
do we ever change created columns ? If not we can exclude from auditing (see @NotAudited)

Reply:
`created_by_mod` & `created_timestamp_mod` removed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/2feddd456da4c714ca6caed9d8207ee8bc40cf46

Reply:
@sullivangareth let's remember to update Nigel's wiki then please

Reply:
Have reinstated auditing on all fields for now, in https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/919ae706dbe69f88ba6404b0bf800b8d624f37d1